Assets/Scripts/UI/SettingsMenu.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using FrameWork;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Audio;
+using Slider = UnityEngine.UI.Slider;
+
+namespace UI
+{
+    public class SettingsMenu : MonoBehaviour
+    {
+        [SerializeField] private AudioMixer audioMixer;
+        [SerializeField] private TMP_Dropdown graphicsDropdown;
+        [SerializeField] private TMP_Dropdown resolutionsDropdown;
+        [SerializeField] private Slider slider;
+        [SerializeField] private Slider progressSlider;
+        [SerializeField] private bool customSettings;
+        [SerializeField] private SceneSwitcher _sceneSwitcher;
+
+        private Resolution[] _resolution;
+
+        private void Awake()
+        {
+            if (progressSlider != null) progressSlider.value = _sceneSwitcher.Progress;
+            LoadSettings();
```

Comment:
```suggestion
            if (progressSlider != null)
                progressSlider.value = _sceneSwitcher.Progress;
            
            LoadSettings();
```
[code convention](https://github.com/Team-Swamp/Project-VooDoo/wiki/Code-convention#if-statements)